examples/nmstate_vars.yaml

```diff
@@ -0,0 +1,11 @@
+---
+nmstate_enabled: false
+nmstate_install_operator: true
+nmstate_catalogsource_image: "" 
+nmstate_upgrade_channel: "stable"
+nmstate_directory: "/tmp/nmstate"
+nmstate_golang_tarball: "https://go.dev/dl/go1.18.6.linux-ppc64le.tar.gz"
+nmstate_e2e: false
+nmstate_e2e_git_repository: ""
```

Comment:
I know this is not needed when e2e flag is false, but can we provide example values here? Empty string in examples does not make sense for people referring it to it. It is same as default variable files.

Reply:
I have added example

Reply:
@varad-ahirwadkar This is an example file, you do not need to comment with "eg.". Directly specify the value.